Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -618,7 +618,7 @@ def audit_bitbucket_repository
 
       new_formula_problem "Uses deprecated mercurial support in Bitbucket" if metadata["scm"] == "hg"
 
-      if metadata["parent"]["full_name"] == "#{user}/#{repo}"
+      if metadata.dig(:parent, :full_name) == "#{user}/#{repo}"
```

Comment:
this is a good fix, but now that I look more closely at this logic, I think the comparison has been wrong. this is comparing the name of the parent repo to the forked repo, which would never be the same. I think we could just see if the `metadata.dig(:parent, :full_name)` is not `nil?` to determine if it's a fork

Reply:
Good call, I like this solution better.